en/frames_autogyro/ThunderFly_AutoG2.md

```diff
@@ -0,0 +1,121 @@
+
+#### 3D printed rotor blades
+Alternatively, it is possible to print blades. They are still under development, but preliminary tests show they are of better quality mostly thanks to their precise shape and absence of longitudinal grooves. However, some of the production processes still need to be tuned.
+
+![Blades assembly](../../assets/airframes/autogyro/auto-g2/modif_blade.png)
+
+#### Balancing
+
+Proper blades balance is very important to minimize vibrations. Blades have to be balanced in such a way, that the center of gravity is located in the middle of the rotor axis.
+
+Printed blades are balanced in the production process and there is no need to further balance them.
+
+### Release device
+
+If you want to launch an autogyro using a winch or if you want to launch it by towing you need to print a release device. It is a small box equipped with servo which pulls out the pin and releases the rope.
+
+The whole part is glued, using a hot-melt adhesive, under the engine on the bottom part of an autogyro’s body. If an autogyro is towed by a rope its engine must not be turned on. It can be handled, for example, by nulling the engine’s output in the transmitter if the release device switch is closed.
+
+![Release device](../../assets/airframes/autogyro/auto-g2/modif_release.png)
+
+## Parts List
+
+### Electronic
+ * Autopilot
+ * GPS
+ * Airspeed sensor
+ * Stronger servos as a substitution for the original ones (optional)
+ * Additional servo for release device (optional)
+
+### Mechanical parts
+
+ * Rotor head Bearing 623 2Z C3
+ * Propeller (APC 10x7)
+ * [Prop adapter](https://mpjet.com/shop/gb/prop-adapters/184-collet-prop-adapter-19-mm-4-mm-shaft-m629-standard.html)
+
+
+### Printable parts
+
+  * Rotor head:
+    * [Pylon end](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/111_1001.stl)
+    * [Pitch part](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/111_1002.stl)
+    * [Roll part](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/111_1003.stl)
+
+  * Rotor:
+    * [center part washer top](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/111_1008.stl)
+    * [center part washer bottom](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/111_1004.stl)
+    * [center plate with deformation zones](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/888_1001.stl)
+    * [washers for setting AoA of blades](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/111_1005.stl)
+    * [Rotor nut](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/888_1002.stl)
+
+  * Rotor blades (optional)
+  * Autopilot holder
+  * [Release device](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/888_1010.stl)
+  * [Front wheels](https://github.com/ThunderFly-aerospace/Auto-G2/blob/master/CAD/stl/888_1011.stl)
+
+### Recommended spare parts
+  * Servos with improved quality

```

Comment:
I guess you're saying that the servos that come with the autopilot aren't great but you could try these other ones? In that case, can you recommend what you would use?

Reply:
Yes, specific servo named, added reason